toolchain/gcc.py

```diff
@@ -52,18 +52,16 @@ def setUp(self):
                              'gmp-devel', 'glibc-devel', 'mpfr-devel',
                              'makeinfo', 'texinfo', 'mpc-devel'])
         else:
-            packages.extend(['glibc-static', 'autogen', 'guile',
-                             'guile-devel', 'libgo', 'libgo-devel',
-                             'libgo-static', 'elfutils-devel',
+            packages.extend(['glibc-static', 'elfutils-devel',
                              'texinfo-tex', 'texinfo', 'elfutils-libelf-devel',
                              'gmp-devel', 'mpfr-devel', 'libmpc-devel',
-                             'gcc-gnat', 'libgnat', 'zlib-devel',
-                             'gettext', 'libgcc', 'libgomp'])
+                             'zlib-devel', 'gettext', 'libgcc', 'libgomp'])
+            if dist.name == 'rhel and (int(dist.version)==8 and int(dist.release >= 6))':
```

Comment:
hey over all it looks good. couple of small suggestions,
1. is this something should be  like below
if dist.name == rhel and '(int(dist.version)==8 and int(dist.release >= 6))': ?
I mean u might have mis-placed the single quot. it might be syntactically wrong. please re-check on this
2. please remove the blank line immediate after the if condition.